### PR TITLE
Add mewConnect access method

### PR DESCRIPTION
**Pull request by SteveMieskoski**

### CODE REVIEW

File: src/pages/Terms.js [hunk 1]
```diff
@@ -153,18 +153,6 @@ const Privacy = () => (
           or quasi-governmental body or regulatory agency, any self-regulatory
           organization.
         </li>
-        <li>
```
Comment (madumas) [inline src/pages/Terms.js:156]: Please remove this change

Comment (SteveMieskoski) [reply src/pages/Terms.js]: I've rectified this branch with master.

**Comment by madumas** [comment]
I can't seem to find @myetherwallet/dai-plugin-mewconnect. Did you publish it?
`error An unexpected error occurred: "https://registry.yarnpkg.com/@myetherwallet%2fdai-plugin-mewconnect: Not found".`

**Comment by SteveMieskoski** [comment]
yeah, I published it.  So, that is strange.  I'll take a look

**Comment by SteveMieskoski** [comment]
Seems like it defaulted or set the package to private.  I updated the permissions so it should find it now.

File: src/utils/constants.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ export const AccountTypes = {
   DCENT: 'dcent',
   METAMASK: 'browser',
   WALLETLINK: 'walletlink',
+  MEWCONNECT: 'MEWconnect',
```
Comment (madumas) [inline src/utils/constants.js:23]: should it be lowecase 'mewconnect'?

File: src/maker.js [hunk 3]
```diff
@@ -71,6 +72,7 @@ export async function instantiateMaker({
       trezorPlugin,
       ledgerPlugin,
       [walletLinkPlugin, walletLinkPluginConfig],
+      [mewconnectPlugin, walletLinkPluginConfig],
```
Comment (madumas) [inline src/maker.js:75]: why the reuse of walletLinkPluginConfig? If the config is necessary, use a different object.

**Comment by SteveMieskoski** [comment]
> I'm not able to connect the wallet. I'm getting the error Unhandled Rejection (Invariant Violation): no factory for type "MEWconnect".
> Plus the text isn't showing up in the dropdown menu.

Fixed the 'no factory type' error, but the text is still not showing up.  Where is that pulled from?  I looked but didn't seem to find it.

**Comment by madumas** [comment]
LGTM. Tested a few operations with MEW Connect successfully

**Pull request merged**